cmd/upgrade/roles/cmd.go

```diff
@@ -1173,3 +1180,15 @@ func checkPolicyAndClusterVersionCompatibility(policyVersion, clusterVersion str
 	}
 	return nil
 }
+
+func printClusterUpgradeInfo(mode string, reporter *rprtr.Object) {
+	if args.isInvokedFromClusterUpgrade && reporter.IsTerminal() {
+		rolesStr := fmt.Sprintf("rosa upgrade roles -c %s --cluster-version=%s --mode=%s",
+			args.clusterID, args.clusterUpgradeVersion, mode)
+		upgradeClusterStr := fmt.Sprintf("rosa upgrade cluster -c %s", args.clusterID)
+		reporter.Infof("Account/Operator Role policies are not valid with upgrade version %s. "+
+			"Run the following command(s) to upgrade the roles and run the upgrade command again:\n\n"+
```

Comment:
Mike, it's clear to me but wonder if this could make it more clear:

"Run the following command(s) to upgrade the roles:\n" +
"\t%s\n\n"+
", then run the upgrade command again:\n" +
"\t%s\n", args.clusterUpgradeVersion, rolesStr, upgradeClusterStr)

Reply:
Done as suggested. Thanks.